### DISCUSSION TITLE
Gopenpgp v2 support after v3 release

### DISCUSSION THREAD

**Discussion by mdosch** [ProtonMail/gopenpgp#270] [Q&A]
Hi,

as afaiu v3 will not be API compatible to v2 (thus the major version bump), I'd like to ask if you will also continue to maintain v2 for a certain period of time after v3 release?
As it is very unlikely that all users of the library will be able to switch to v3 immediately it would be nice if there would be a grace period where v2 still gets important Bugfixes.

Best regards,
Martin

**Comment by lubux** [comment]
Hi,

Yes, we will still support v2. There will also be a v2 release, which includes the crypto refresh.

- Lukas

**Comment by mdosch** [comment]
Thank you very much, this is good news. 🙂

Am 10. Februar 2024 15:12:05 UTC schrieb Lukas Burkhalter ***@***.***>:
>Hi,
>
>Yes, we will still support v2. There will also be a v2 release, which includes the crypto refresh.
>
>- Lukas 
>
